src/functions.js

```diff
@@ -0,0 +1,38 @@
+const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
+/**
+ *
+ * @param {string} searchWord
+ * @returns {any[]}
+ */
+function filterThroughEpisodes(searchWord) {
+    return gameOfThronesEpisodes.filter((episode) => {
+        const nameMatch = episode.name
+            .toLowerCase()
+            .includes(searchWord.toLowerCase());
+        const summaryMatch = episode.summary
+            .toLowerCase()
+            .includes(searchWord.toLowerCase());
+
+        return nameMatch || summaryMatch;
+    });
+}
+
+function generateEpisodeCode(episode) {
+    const paddedSeason = String(episode.season).padStart(2, "0");
+    const paddedEpisode = String(episode.number).padStart(2, "0");
+
+    return `S${paddedSeason}E${paddedEpisode}`;
+}
+
+function removePTags(episodes) {
```

Comment:
a bit unsure about what this one does, especially on line 29. just want something like a jsdoc to further explain what the function is meant to do